src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -232,6 +232,8 @@ private function joinRelations(QueryBuilder $queryBuilder, QueryNameGeneratorInt
 
             $this->joinRelations($queryBuilder, $queryNameGenerator, $mapping['targetEntity'], $forceEager, $fetchPartial, $associationAlias, $options, $normalizationContext, 'leftJoin' === $method, $joinCount, $currentDepth);
         }
+
+        $queryBuilder->getQuery()->getResult();
```

Comment:
Mhhh but this will execute the query multiple time right? I don't get the point.

May we have a test illutrating the bug in https://github.com/api-platform/api-platform/issues/799 ?

Reply:
@soyuka that will not affect performance. The whole process is described [here](http://ocramius.github.io/blog/doctrine-orm-optimization-hydration/)

Reply:
I assume you're talking about the comment saying: `// result is discarded (this is just re-hydrating the collections)`. May you add a comment like this because it's really confusing if you're not familiar with doctrine internals...